tendrl/ceph_integration/manager/cluster_monitor.py

```diff
@@ -350,14 +350,23 @@ def on_sync_object(self, data):
                 ) else None,
                 now(), sync_object, self._manager.cluster_id)
             if sync_type.str == "osd_map":
+                util_data = self._get_utilization_data()
                 for raw_pool in sync_object.get('pools', []):
                     LOG.info("Updating Pool %s" % raw_pool['pool_name'])
+                    for pool in util_data['pools']:
+                        if pool['name'] == raw_pool['pool_name']:
+                            max_avail = pool['stats']['max_avail']
+                            pool_used = pool['stats']['bytes_used']
+                            pcnt = (pool_used * 100) / (pool_used + max_avail)
```

Comment:
Ceph uses a different logic to do this calculation in case of multiple profiles, so it might be better to get this directly from the normal ceph df command and parse it rather than calculating it from output of ceph df command